templates/ts-apollo-mongodb-backend/src/index.ts

```diff
@@ -1,36 +1,35 @@
 // eslint-disable-next-line @typescript-eslint/no-require-imports
-require('dotenv').config()
-import path from 'path'
-import { ApolloServer } from "apollo-server-express"
-import { buildGraphbackAPI } from 'graphback'
-import { loadSchemaSync } from '@graphql-tools/load'
-import { GraphQLFileLoader } from '@graphql-tools/graphql-file-loader'
```

Comment:
These can also be removed from `package.json`

Reply:
Good catch.